### DISCUSSION TITLE
Does Next.js image component handle converting WEBP to JPG if device doesn't support WEBP?

### DISCUSSION THREAD

**Discussion by koji98** [vercel/next.js#40817] [Help]
Hello, just wondering if the Next image component also handles re-formatting images to an older format (JPG/PNG) if a user's device doesn't support newer ones (WEBP/AVIF)? I'd like to convert some images on my app from JPG to WEBP, but don't want to risk messing up the UX of user's on older devices/browsers.

**Comment by cat-michael-gwynne** [comment]
I'm using lh3.googleusercontent and those URLs accept a load of image options: https://github.com/votemike/lh3-googleusercontent-url-builder/blob/main/index.d.ts
I would love to use next/image but instead have to use a custom made `<picture>` component to support both webp and jpg.
